notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb

```diff
@@ -122,6 +122,8 @@
     "from azureml.core.image import ContainerImage\n",
```

Comment:
<p>This is so cool man, I don't think there something like this anywhere</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/735/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Thanks! That estimation was hidden in docs (at the link I referenced), but it **is** really useful. Was wondering if this should actually be a function in common or in azureml that we could just import.</p>

Reply:
I guess if the estimation is only related to azureml, it should go in azureml folder

Reply:
<p>ok, moved these, renamed them, and added tests.</p>